bedRModv2.tex

```diff
@@ -0,0 +1,393 @@
+
+\section{Examples}
+
+\subsection[title]{Example bedRMod file from the \acs{bedRMod} and related specifications\footnote{\url{https://github.com/dieterich-lab/euf-specs/examples/bedrmod/example.bedrmod}}}\label{sec:example-bedrmod}
+
+\begin{verbatim}
+#fileformat=bedRModv2
+#organism=9606
+#modification_type=RNA
+#modification_names=20607:m5C:C,21891:m6A:A
+#assembly=GRCh38
+#annotation_source=Ensembl
+#annotation_version=110
+#sequencing_platform=Illumina NovaSeq 6000
+#basecalling=
+#bioinformatics_workflow=workflow:https://github.com/XXX
+#experiment=https://doi.org/10.XXX
+#external_source=SRA:PRJNAXXXXXX,GEO:GSEXXXXXX
+#chrom	chromStart	chromEnd	name	score	strand	thickStart	thickEnd	itemRgb	coverage	frequency
+1	1391918	1391919	20607	0	-	1391918	1391919	0,0,0	42	42.56
+2	8878712	8878713	20607	0	-	8878712	8878713	0,0,0	318	44.23
+3	11980442	11980443	21891	0	+	11980442	11980443	0,0,0	111	56.20
+4	17054111	17054112	20607	0	-	17054111	17054112	0,0,0	40	34.03
+5	23691799	23691800	21891	0	+	23691799	23691800	0,0,0	352	27.33
+\end{verbatim}
+
+\section{Recommended practice for the \acs{bedRMod} format}
+
+\subsection{\acs{bedRMod} extension}
+The~\textbf{file} extension is~\texttt{.bedrmod},~\texttt{.bedmethyl}, or~\texttt{.bed}. Since the \ac{BED} format prohibits~\textbf{BED11}, there should be little confusion in general, but when~\textbf{custom field}s are defined, it is recommended to use the~\texttt{.bedrmod} or~\texttt{.bedmethyl} extension. \Ac{bedRMod} formalizes the ENCORE bedMethyl format\footnote{Description of bedMethyl file, \url{https://www.encodeproject.org/data-standards/wgbs}} for \emph{RNA modification}s.
+
+\subsection{Mandatory \acs{bedRMod} header fields}
+These~\textbf{header field}s are not free text, and must conform to a controlled vocabulary.
+\begin{itemize}
+\item \textsf{fileformat}: A valid version of this specification, including the format name, \textit{e.g.}~\texttt{bedRModv2}. 
+
+\item \textsf{modification\textunderscore type}: A valid RNA type\footnote{For example RNA or mRNA, tRNA, or rRNA. A next version of this specification should prescribe a controlled vocabulary by providing a reference to an established RNA ontology.}.
+
+\item \textsf{modification\textunderscore names}: A comma-separated dictionary mapping in the form name:MODOMICS \emph{short name}:reference or unmodified base, where name corresponds to the~\textsf{name}~\textbf{field} of a~\textbf{data line}, MODOMICS \emph{short name} to the corresponding short name of the MODOMICS nomenclature, and reference or unmodified base is the unmodified base as reported by the sequencing instrument for the top strand, as described in the SAMtags, except that `N' is not allowed (since it can be used to match any base). A value is also required when using MODOMICS \emph{short names}, \textit{e.g.}~\texttt{m6A:m6A:A}. All modifications present in the~\textbf{file} must be included, in a comma-separated list of items.
```

Comment:
It's a complete nit-pick, but you may want to change the language from "reported by the sequencing instrument for the top strand" to "primary sequence base". The fact that the MM tags allow you to report modifications on double stranded DNA by using `G-m` to mean 5mC on the (-)-strand is no longer important here, what's important to say is that "code `<code>` is a modification to `<base>` primary base".